portfolio/src/main/webapp/mocha_tests/calendar_test.js

```diff
@@ -0,0 +1,131 @@
+var assert = require('assert');
+var calendar = require('../calendar.js'); 
+
+/**
+ * In general, these tests are for date/time string parsing and validity.
+ * Because these values come from elements on the HTML file that are
+ * elements of type time or date, the HTML element already checks for 
+ * some invalid input. 
+ * Specifically, the numbers inputted for month, date, hour, and minute
+ * that come from these HTML elements are valid numbers for date and time.
+ * For example, 20 is not a valid month, and 70 is not a valid minute.  
+ * Invalid inputs in this aspect will not be tested in the following tests. 
+ */
+
+describe('#parseTodayString', function () {
```

Comment:
Similar comment here about constructTodayString instead of parseTodayString

Reply:
Updated.